website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -61,16 +61,15 @@ Read the [operator scope][operator_scope] documentation on how to run your opera
 
 ### Multi-Group APIs
 
-Before creating an API and controller, consider if your operator's API requires multiple [groups][API-groups].
-If yes then add the line `multigroup: true` in the `PROJECT` file which should look like the following:
+Before creating an API and controller, consider if your operator's API requires multiple [groups][API-groups]. Then to change the layout of your project to support Multi-Group run the command `operator-sdk edit --multigroup=true`. It will update the `PROJECT` file which should look like the following:
```

Comment:
```suggestion
Before creating an API and controller, consider if your operator requires multiple API [groups][api-groups]. Then to change the layout of your project to support multi-group run the command `operator-sdk edit --multigroup=true`. It will update the `PROJECT` file which should look like the following:
```